From 6b6580d3d60b0a773feb84659ce93e058b1b7831 Mon Sep 17 00:00:00 2001
From: Lawrence Rust <lvr@softsystem.co.uk>
Date: Thu, 31 May 2012 10:06:36 +0200
Subject: [PATCH 091/227] Themes: Fix font weight in MythCenter-wide 'program
 info' window

Fix an error with the "program_info" window which was attempting
to set font weight within a textarea.

Signed-off-by: Lawrence Rust <lvr@softsystem.co.uk>
---
 mythtv/themes/MythCenter-wide/osd.xml |    9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/mythtv/themes/MythCenter-wide/osd.xml b/mythtv/themes/MythCenter-wide/osd.xml
index 2ac0351..ab7b669 100644
--- a/mythtv/themes/MythCenter-wide/osd.xml
+++ b/mythtv/themes/MythCenter-wide/osd.xml
@@ -11,6 +11,9 @@
         <fontdef name="medium" from="small">
             <pixelsize>28</pixelsize>
         </fontdef>
+        <fontdef name="medium_bold" from="medium">
+            <weight>bold</weight>
+        </fontdef>
         <fontdef name="large" from="small">
             <pixelsize>34</pixelsize>
             <weight>bold</weight>
@@ -26,14 +29,12 @@
         </imagetype>
 
         <textarea name="channame">
-            <font>medium</font>
-            <weight>bold</weight>
+            <font>medium_bold</font>
             <area>20,20,860,40</area>
             <align>left,bottom</align>
         </textarea>
         <textarea name="channum">
-            <font>medium</font>
-            <weight>bold</weight>
+            <font>medium_bold</font>
             <area>880,20,280,40</area>
             <align>right,bottom</align>
         </textarea>
-- 
1.7.9.5

